app/ui/LoadingSpinner.tsx

```diff
@@ -0,0 +1,36 @@
+import { ReactElement, ReactNode, useState } from "react";
+import { loadingRed } from "./colourLibrary";
+import styled, { keyframes } from "styled-components";
+
+export function LoadingSpinner(props: {
+  width?: number;
+  height?: number;
+}): ReactElement {
+  const { height, width } = props;
+  return (
+    <StyledDiv
+      style={{
+        height,
+        width,
+      }}
+    ></StyledDiv>
```

Comment:
if there's nothing inside you can make the tags self-closing

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/25/commits/0fd686f6fbbf430bda844fcf91daa4366b5d4486